.gitignore

```diff
@@ -30,6 +30,10 @@ motd.txt*
 /logs
 /ongoing_tables
 
+# Development files
+*.code-workspace
+package-lock.json
```

Comment:
add this is a bug because we use yarn, right?

Reply:
I run the build as instructed for linux.  had to `npm install` first to set the lock file.

Reply:
thats old info, you shouldn't ever have to invoke npm

Reply:
on this page it doesn't say anything about doing `npm install` https://github.com/Hanabi-Live/hanabi-live/blob/main/docs/INSTALL.md

Reply:
That being said, it did not work without first running `npm install` because it complained about a missing lockfile

Reply:
its because of this bug https://github.com/Hanabi-Live/hanabi-live/commit/37631cb77966973327eb904d113dce2df4fe1083